gcloud/pubsub/client.py

```diff
@@ -50,6 +50,12 @@ def list_topics(self, page_size=None, page_token=None):
         See:
         https://cloud.google.com/pubsub/reference/rest/v1/projects.topics/list
 
+        Example:
+
+        .. literalinclude:: pubsub_snippets.py
+           :start-after: [START client_list_topics]
+           :end-before: [END client_list_topics]
```

Comment:
> So what is the solution?

Just tell people to RTFM.?  ISTM that the win of having the snippets run as real system tests trumps having examples bitrotting in place in the docstrings.